README.md

```diff
@@ -94,24 +50,11 @@ If you have the Git credential manager core or other credential helpers installe
 
 1.  Create a writable folder to cache 3rd Party dependencies. You can also use this to store other redistributable SDKs.
     
-    > For the 0.5 branch - Create an empty text file named `3rdParty.txt` in this folder, to allow a legacy CMake validator to pass
-
 1.  Install the following redistributables to the following:
     - Visual Studio and VC++ redistributable can be installed to any location
     - CMake can be installed to any location, as long as it's available in the system path
-    - WWise can be installed anywhere, but you will need to set an environment variable for CMake to detect it:  `set LY_WWISE_INSTALL_PATH=<path to WWise>`
+    - (Optional) WWise can be installed anywhere, but you will need to set an environment variable for CMake to detect it:  `set LY_WWISE_INSTALL_PATH=<path to WWise>`
```

Comment:
Nit: WWise -> Wwise

Reply:
I pushed another update to mine, but it looks like the files should be able to auto merge.